docs/ops/deployment/yarn_setup.md

```diff
@@ -268,10 +268,27 @@ To launch an application in [Application Mode]({% link ops/deployment/index.md %
 As an example, the command to specify the memory sizes of the JM and the TM, looks like:
 
 {% highlight bash %}
-./bin/flink run-application -t yarn-application -Djobmanager.memory.process.size=2048m -Dtaskmanager.memory.process.size=4096m  ./examples/batch/WordCount.jar
+./bin/flink run-application -t yarn-application \
+-Djobmanager.memory.process.size=2048m \
+-Dtaskmanager.memory.process.size=4096m \
+./examples/batch/WordCount.jar
+
+{% endhighlight %}
+
+For a look at the available configuration options, you can have a look [here]({% link ops/config.md %}). To unlock
+the full potential of the application mode, consider using it with the `yarn.provided.lib.dirs` configuration option
+and pre-upload your application jar to a location accessible by all nodes in your cluster. In this case, the 
+command could look like: 
+
+{% highlight bash %}
+./bin/flink run-application -t yarn-application \
+-yD yarn.provided.lib.dirs="hdfs://myhdfs/my-remote-flink-dist-dir" \
```

Comment:
I think it should be `-D` not `-yD`.

Reply:
Of course, and also no space. The problems of copy-paste without re-reading.